interfaces/cython/cantera/thermo.pyx

```diff
@@ -379,6 +379,22 @@ cdef class ThermoPhase(_SolutionBase):
         :param loglevel:
             Set to a value > 0 to write diagnostic output.
             """
+        # check for old parameters and warn the user
+        if 'maxsteps' in **kwargs:
```

Comment:
Currently there is a syntax error here (which caused the CI checks to fail): when checking the contents of a dictionary, the `**` that unpack it shouldn't be there. I.e. 

```python
if 'maxsteps' in kwargs:
    ...
``` 
should work. Before resubmitting, I'd also recommend running `scons build` and `scons test` on your own machine, as it will catch those glitches.